app/controllers/providers_controller.rb

```diff
@@ -26,6 +28,8 @@ def show
   end
 
   def details
+    redirect_to_new_publish_url_details_provider_recruitment_cycle_path if FeatureService.enabled?("new_publish.about_your_org")
```

Comment:
I think we should do the same for `contact`, `about`, and `update` as well (ie—all actions that have now been reimplemented in New Publish). Also, `Providers::VisasController`.

Reply:
Oh right 😂